mass_editing/models/mass_object.py

```diff
@@ -10,7 +10,7 @@ class MassObject(models.Model):
     _name = "mass.object"
     _description = "Mass Editing Object"
 
-    name = fields.Char('Name', required=True, select=1)
+    name = fields.Char('Name', required=True, index=1)
```

Comment:
remove 'Name' parameter, not necesary on v10 after the fact it is taking field name on Camel Case if no string attribute (just for this feald no refactor all please).